python/tvm/relay/op/nn/_nn.py

```diff
@@ -157,8 +157,9 @@ def convert_conv2d(attrs, inputs, tinfos, desired_layouts):
         return relay.nn.conv2d(data, weight, **new_attrs)
     elif desired_data_layout == 'NHWC':
         # Check for depthwise convolution.
-        if is_depthwise_conv2d(data.shape, attrs['data_layout'], weight.shape,
-                               attrs['kernel_layout'], attrs['groups']):
+        if is_depthwise_conv2d(data.checked_type.shape, attrs['data_layout'],
```

Comment:
It might worth putting an assert to make sure `*.checked_type.shape` exist at this point.

Reply:
I think it's ok to leave without an assert, getting `checked_type` will result in a check for its existence. See: https://github.com/apache/incubator-tvm/blob/0465ffda26a4e9d64944f01b33b566b6663d3afe/python/tvm/ir/expr.py#L36